api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,104 @@ const (
+	// When omitted, this means no opinion, and the platform is left to choose a reasonable default.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=86400
+	// +optional
+	DelayAfterAddSeconds *int32 `json:"delayAfterAddSeconds,omitempty"`
+
+	// delayAfterDeleteSeconds sets how long after node deletion, scale down evaluation resumes, defaults to scan-interval
+	// It must be between 0 and 86400.
+	// When set to 0, this means scale down evaluation will resume immediately after node deletion, without any delay.
+	// When omitted, this means no opinion, and the platform is left to choose a reasonable default.
+	// The default value is 60.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=86400
+	// +kubebuilder:default=60
+	// +optional
+	DelayAfterDeleteSeconds *int32 `json:"delayAfterDeleteSeconds,omitempty"`
+
+	// delayAfterFailureSeconds sets how long after a scale down failure, scale down evaluation resumes
+	// It must be between 0 and 86400.
+	// When set to 0, this means scale down evaluation will resume immediately after a scale down failure, without any delay.
+	// When omitted, this means no opinion, and the platform is left to choose a reasonable default.
+	// The default value is 60.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=86400
+	// +kubebuilder:default=60
+	// +optional
+	DelayAfterFailureSeconds *int32 `json:"delayAfterFailureSeconds,omitempty"`
+
+	// unneededDurationSeconds establishes how long a node should be unneeded before it is eligible for scale down in seconds
+	// It must be between 0 and 86400.
+	// When set to 0, this means nodes will be considered for scale down immediately when they become unneeded, without any waiting period.
+	// When omitted, this means no opinion, and the platform is left to choose a reasonable default.
+	// The default value is 600.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=86400
+	// +kubebuilder:default=600
+	// +optional
+	UnneededDurationSeconds *int32 `json:"unneededDurationSeconds,omitempty"`
+
+	// utilizationThreshold determines the node utilization level, defined as sum of requested resources divided by capacity, below which a node can be considered for scale down
+	// The value represents a percentage from 0 to 100.
+	// When set to 0, this means nodes will be considered for scale down even if they are completely idle (0% utilization).
+	// When set to 100, this means nodes will only be considered for scale down when they are fully utilized (100% utilization).
+	// A value between 0 and 100 represents the utilization threshold below which a node can be considered for scale down.
+	// When omitted, this means no opinion, and the platform is left to choose a reasonable default.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=100
+	// +optional
+	UtilizationThreshold *int32 `json:"utilizationThreshold,omitempty"`
```

Comment:
the valid values for this are from `0.0` to `1.0`, with a default of `0.5`. if we follow the convention from the CAO, this should be a string not an int32.

Reply:
Thanks, change to UtilizationThresholdPercent